### PR TITLE
Add basic snapshot summary page

### PR DESCRIPTION
**Pull request by mmmmillar**
**What**  
Add a basic summary page

**Why**  
So we can see the details of a snapshot

Screenshot 2020-06-22 at 10 26 31

### CODE REVIEW

File: components/Form/AccordionItem/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import '../../../pages/stylesheets/vars.scss';
+
+.lbh-accordion__section-header {
+  color: $hackney-green !important;
```
Comment (kosiakkatrina) [inline components/Form/AccordionItem/index.module.scss:4]: Is there a way to do it without important? 🤔

File: public/js/govuk.js [hunk 2]
```diff
@@ -819,10 +819,13 @@ Accordion.prototype.initControls = function () {
   this.$openAllButton.setAttribute('type', 'button');
 
   // Create control wrapper and add controls to it
-  var accordionControls = document.createElement('div');
-  accordionControls.setAttribute('class', this.controlsClass);
+  // var accordionControls = document.createElement('div');
+  // accordionControls.setAttribute('class', this.controlsClass);
+  // accordionControls.appendChild(this.$openAllButton);
+  // this.$module.insertBefore(accordionControls, this.$module.firstChild);
```
Comment (kosiakkatrina) [inline public/js/govuk.js:825]: Do we need the comments?

File: pages/snapshots/[id].js [hunk 3]
```diff
@@ -6,24 +7,90 @@ import { Button, TextArea } from 'components/Form';
...
   if (loading) {
     return <p>Loading...</p>;
   }
 
+  const [editSnapshot, setEditSnapshot] = useState(
+    snapshot.assets.length === 0 &&
+      snapshot.vulnerabilities.length === 0 &&
+      !snapshot.notes
+      ? true
+      : false
```
Comment (kosiakkatrina) [inline pages/snapshots/[id].js]: Can we omit these?

**Review by kosiakkatrina — APPROVED** [review]
Looks good, would be nice if there were cypress tests

**Review by t-davies — APPROVED** [review]

File: public/js/govuk.js (hunk 2 above)
Comment (mmmmillar) [reply public/js/govuk.js:825]: It's not our code, I'm gonna leave them there

File: components/Form/AccordionItem/index.module.scss (hunk 1 above)
Comment (mmmmillar) [reply components/Form/AccordionItem/index.module.scss:4]: I don't think it matters at all as the css is scoped

File: public/js/govuk.js (hunk 2 above)
Comment (kosiakkatrina) [reply public/js/govuk.js:825]: Ah, ok, I thought it might not be thank you!

File: components/Form/AccordionItem/index.module.scss (hunk 1 above)
Comment (kosiakkatrina) [reply components/Form/AccordionItem/index.module.scss:4]: Fair

**Comment by mmmmillar** [comment]
> Looks good, would be nice if there were cypress tests

one thing at a time

**Pull request merged**